lib/storages/type-metadata.storage.ts

```diff
@@ -33,6 +39,10 @@ export class TypeMetadataStorageHost {
   ) {
     return this.properties.filter(belongsToClass);
   }
+
+  getVirtualsMetadataByTarget<TClass>(targetFilter: Type<TClass>) {
```

Comment:
Isn't it better to add `virtuals` to `SchemaMetadata` and then create method `getClassVirtualsByPredicate` like the one below ?